### ISSUE TITLE
Lack of Examples

### ISSUE THREAD

**Issue by frankcash** [jonbo/node-steam-webapi#1]
Could you give examples such as getting games from dota2 with certain heroes?

Or just more examples in general?

**Comment by ghost** [comment]
I know this is **really** late, but just in case someone new sees this:

You can find the whole list of api calls here: https://github.com/jonbo/node-steam-webapi/blob/master/api_method_list.md

**Comment by demipixel** [comment]
Yeah another year later, but be warned the examples on the front lie. You must do `Steam.INTERFACES[interface].func()`, not `Steam.func()`. Best to log out Steam to see the interface.
